src/main/java/com/synopsys/integration/jira/common/model/response/VersionResponseModel.java

```diff
@@ -38,16 +42,20 @@ public VersionResponseModel() {
     public VersionResponseModel(
         String self,
         String id,
-        String name,
+        String description, String name,
         boolean archived,
         boolean released,
-        String projectId
+        String releaseDate, boolean overdue, String userReleaseDate, String projectId
```

Comment:
I'm guessing these were generated, but it might be good to consistently space them.